panel/src/components/Layout/Item.vue

```diff
@@ -221,7 +221,8 @@ export default {
 .k-list-item .k-item-figure {
   width: 38px;
   border-start-start-radius: var(--rounded-sm);
-  border-start-end-radius: var(--rounded-sm);
+  border-end-start-radius: var(--rounded-sm);
+  box-shadow: var(--shadow);
```

Comment:
The added box shadow makes it look like the image sits above the rest of the list row. I am not sure if I like it or not, but it's a bit of a change.

Reply:
👍 Then let's continue as before without shadow.